src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -362,21 +363,31 @@ def consRecordAge(self, rse):
         :return:    rse or raises MSUnmergedPlineExit
         """
         rseName = rse['name']
+
+        if rseName not in self.rseConsStats:
+            msg = "RSE: %s Missing in stats records at Rucio Consistency Monitor. " % rseName
+            msg += "Skipping it in the current run."
+            self.logger.info(msg)
+            raise MSUnmergedPlineExit(msg)
+
         isConsDone = self.rseConsStats[rseName]['status'] == 'done'
         isConsNewer = self.rseConsStats[rseName]['end_time'] > self.rseTimestamps[rseName]['prevStartTime']
         isRootFailed = self.rseConsStats[rseName]['root_failed']
 
         if not isConsNewer:
-            msg = "RSE: %s With old consistency record in Rucio Consistency Monitor. Skipping it in the current run."
-            self.logger.info(msg, rseName)
+            msg = "RSE: %s With old consistency record in Rucio Consistency Monitor. " % rseName
+            msg += "Skipping it in the current run."
+            self.logger.info(msg)
             raise MSUnmergedPlineExit(msg)
         if not isConsDone:
-            msg = "RSE: %s In non-final state in Rucio Consistency Monitor. Skipping it in the current run."
-            self.logger.info(msg, rseName)
+            msg = "RSE: %s In non-final state in Rucio Consistency Monitor. " % rseName
+            msg += "Skipping it in the current run."
+            self.logger.info(msg)
```

Comment:
I'd suggest to make this one a warning record as well.

Reply:
agreed